src/main/resources/migrations.xml

```diff
@@ -528,4 +528,19 @@
             UPDATE invocation_output_column SET class = 'java.time.Instant' WHERE class = 'java.time.LocalDateTime' OR class = 'java.time.LocalDate';
         </sql>
     </changeSet>
+    <changeSet id="9" author="Do Bao Khanh &lt;khanh.do@jubic.fi&gt;">
+        <addColumn tableName="external_client" >
+            <column name="description" type="text">
+                <constraints nullable="true" />
```

Comment:
Please no nullable strings

Reply:
Remove nullable contraint. Change to `<column name="description" type="string">`